src/style/app.module.css

```diff
@@ -428,20 +433,28 @@
 .dropdownItem:hover,
 .dropdownItem:focus,
 .dropdownItem:active {
-  background-color: var(
-    --dropdown-hover-color,
-    var(--grey-border-box-color)
-  ) !important;
-  color: var(--brown-color) !important;
+  background-color: var(--holidays-card-background-color) !important;
+  color: var(--black-color) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`